stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/extensions/ChatClient.kt

```diff
@@ -0,0 +1,33 @@
+package io.getstream.chat.android.offline.extensions
```

Comment:
Let's add:
```kotlin
@file:JvmName("ChatClientExtensions")
```

In order to use it in Java like this:
```java
public void requestMembers() {
            ChatClient chatClient = ChatClient.instance();
            ChatClientExtensions.requestMembers(...);
        }
```

Reply:
Done.